Makefile

```diff
@@ -43,7 +43,7 @@ motor: $(OBJS)
 	$(CXX) -o motor $^ $(LDFLAGS) $(LIBS)
 
 %.o: %.cc
-	$(CXX) -DSHARE_DIR=\"$(datadir)/motor\" $(INCLUDES) $(CPPFLAGS) $(CXXFLAGS) -c -o  $@ $<
+	$(CXX) -DSHARE_DIR="\"$(datadir)/motor\"" $(INCLUDES) $(CPPFLAGS) $(CXXFLAGS) -c -o  $@ $<
```

Comment:
why's that necessary? it builds fine without the change here.
also this has nothing to do with the commit message

Reply:
Defines replace text directly. It was putting /usr/share/motor in the code
Instead of the string literal “/usr/share/motor”

Reply:
I had a bunch of commits that had multiple different things in one commit

Reply:
It defaults to /usr/local/share so it appeared to still work but when I had changed prefix to /usr it still said /usr/local. Putting the quotes around that fixed that

Reply:
if the macro would expand to /usr/share without quote where a string is expected this would simply break the compilation. so it has nothing to do with your /usr vs /usr/local problem.

Reply:
Well for some reason changing prefix to /usr in the makefile still put /usr/local/share/motor in the share define until I put those quotes there. Motor wasn’t finding the templates and I was trying to figure out why

Reply:
This is a problem because I couldn’t set prefix if I wanted motor to be able to find its templates

Reply:
I can remove them but it will break whenever I try to package motor

Reply:
nvm im just dumb and something else fixed it. it seems to make no difference now.